Content.Server/Radio/EntitySystems/RadioSystem.cs

```diff
@@ -78,22 +81,15 @@ public void SendRadioMessage(EntityUid messageSource, string message, RadioChann
         if (!_messages.Add(message))
             return;
 
-        var name = TryComp(messageSource, out VoiceMaskComponent? mask) && mask.Enabled
-            ? mask.VoiceName
-            : MetaData(messageSource).EntityName;
+        var evt = new TransformSpeakerNameEvent(messageSource, MetaData(messageSource).EntityName);
+        RaiseLocalEvent(messageSource, evt);
 
-        name = FormattedMessage.EscapeText(name);
+        var name = FormattedMessage.EscapeText(evt.VoiceName);
 
-        SpeechVerbPrototype speech;
-        if (mask != null
-            && mask.Enabled
-            && mask.SpeechVerb != null
-            && _prototype.TryIndex<SpeechVerbPrototype>(mask.SpeechVerb, out var proto))
-        {
+        var speech = _chat.GetSpeechVerb(messageSource, message);
```

Comment:
Looks like it transformed the radio's speech verb before and doesn't now? Though this might make more sense since it always whispers.

Reply:
You overestimate the chat system! I think it sends the whisper separately and in the whisper function the verb is always whisper.

Also I think it does change the verb there? I just checked again and it appears to be working!
![image](https://github.com/user-attachments/assets/517490c3-b814-4e35-9f7f-ba57bd387e89)